content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to nonhuman animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to nonhuman animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that falsely identifies one group as deservedly superior to another.   
+
+and  
```

Comment:
Maybe just merge these two blockquotes back together if they need to be treated as one response? (i.e. remove the `and` and the space between them)

Reply:
The problems with the 'ors' but I'm looking at that bit

Reply:
Version 207.6, see what you think.